java/src/com/github/bazel_contrib/contrib_rules_jvm/junit5/README.md

```diff
@@ -0,0 +1,280 @@
+# JUnit 5 Test Runner
+
+This package contains a JUnit5 test runner. The design goals for this are:
+
+* To not leak any classes on to the test's classpath
+  * This implies only using classes from the JRE or JUnit5 base classes
+* To provide test logs...
+  * ... that can be consumed by the Bazel IJ plugin
+  * ... that are as similar to those generated by Maven and Gradle as possible
+
+The requirements about test logs are to support running tests via the IJ plugin,
+and to allow test results to be ingested by existing systems that process the 
+JUnit4 XML output. For the sake of argument, we shall be attempting to use the
+[JUnit XML XSD][xsd] provided by the JUnit team.
+
+For reference, the JUnit XSD suggests that there be a top-level `testsuites` 
+tag. This can contain any number of `testsuite` tags. Each of these 
+`testsuite` tags may only contain any number of nested `testcase` tags. The 
+model does not allow nesting of `testsuite` tags within `testsuite` tags. 
+
+## Test log formats
+
+JUnit5 has a number of different ways of running and not running tests, so a 
+quick review of how each of the existing test frameworks. For each of these 
+we'll consider the various following combinations:
+
+* "Simple": A regular test case, using no special features
+* "Suite": A test suite, created using the `@Suite` annotation and containing 
+  two test classes
+* "Dynamic" A dynamic test using the `@TestFactory` annotation
+* "Parameterised" A parameterised test case
+* "Pathological" A parameterised test case where the test method is disabled 
```

Comment:
Could you rename `DisabledSkippedParameterisedTest` in the comparative-tests to `PathologicalTest`, or rename it in this file to `DisabledSkippedParameterisedTest`? All of the others line up nicely, but I had t odo some reading around to understand what example test this was referring to.